cpp/ql/src/semmlecode.cpp.dbscheme

```diff
@@ -1041,6 +1048,11 @@ exprconv(
 
 compgenerated(unique int id: @element ref);
 
+synthetic_destructor_call(
```

Comment:
Please add a comment on this table, especially to explain that the `i` go backwards. Ideally the comment would be adapted from the QLDoc on `getSynthesisedDestructorCallAfterNode`, but that would require the class names to be replaced with dbtypes, and I can understand if that's too much work or reduces clarity too much.

Reply:
For now, I've just gone with a shorter qldoc. I might want to revisit this in the future.